From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: deepak1556 <hop2deep@gmail.com>
Date: Fri, 9 Aug 2024 22:39:47 +0900
Subject: build: allow electron mojom interfaces to depend on blink
 mojom_platform

Needed for:

1) //electron/shell/common:plugin
2) //electron/shell/common:web_contents_utility

diff --git a/content/public/common/BUILD.gn b/content/public/common/BUILD.gn
index 33440fbace3d6d9cbbe74c4ce0ccf7b6137b43e5..e92dac385501107848e9937cdc145ed0d2ec3bb7 100644
--- a/content/public/common/BUILD.gn
+++ b/content/public/common/BUILD.gn
@@ -363,6 +363,8 @@ mojom("interfaces") {
     "//content/common/*",
     "//extensions/common:mojom",
     "//extensions/common:mojom_blink",
+    "//electron/shell/common:plugin",
+    "//electron/shell/common:web_contents_utility",
   ]
 
   sources = [
